examples/seismic/tutorials/14_creating_synthetics.ipynb

```diff
@@ -30,6 +30,7 @@
     "    import gempy as gp\n",
     "except ModuleNotFoundError:\n",
     "    # Need to install these\n",
+    "    ! pip install aiohttp==3.7.1\n",
```

Comment:
how does that help?

Reply:
Any chance you checked whether now, we can have working version without installing specific gempy and specific pyvista? @EdCaunt ?